src/plugins/impressConsole/impressConsole.js

```diff
@@ -16,6 +16,12 @@
 ( function( document, window ) {
     'use strict';
 
+    var notes = window.document.getElementsByClassName( 'notes' );
+
+    for ( let items in Object.entries( notes ).values ) {
+        items.style.display = 'none';
+    }
+
```

Comment:
This seems to be a bug fix in its own right! Can you please file it as a separate PR and undo in this PR.